dashboards/MongoDB/MongoDB_Cluster_Summary.json

```diff
@@ -3799,7 +3799,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "max by (set) (max(max_over_time(mongodb_mongod_replset_member_replication_lag{cluster=\"$cluster\"}[$interval]) > 0) by (service_name,set) or max(max_over_time(mongodb_mongod_replset_member_replication_lag{cluster=\"$cluster\"}[5m]) > 0) by (service_name,set))",
+          "expr": "max by (set) (max(max_over_time(mongodb_mongod_replset_member_replication_lag{cluster=\"$cluster\",state!="ARBITER"}[$interval]) > 0) by (service_name,set) or max(max_over_time(mongodb_mongod_replset_member_replication_lag{cluster=\"$cluster\",state!="ARBITER"}[5m]) > 0) by (service_name,set))",
```

Comment:
state!=\"ARBITER\"

Escape character( \ ) is missing for string quotation( " ).